### DISCUSSION TITLE
How to increase CPU or reserve more than One for dokku postgres db ?

### DISCUSSION THREAD

**Discussion by dufok** [dokku/dokku#5681] [Q&A] [answered]
**How to increase CPU or reserve more than One for dokku postgres db ?**

I have limits on CPU usage of postgres container and can't reserve more with _dokku resource:limit_ ...

Is it option for that ? or i need downgrade to docker container ?

**Comment by josegonzalez** [comment, marked as answer]
The service plugins - in this case postgres - do not support limiting or reserving cpu resources, only memory.

**Reply by dufok** [reply]
Interesting ! Please provide more info about this option !!! ( memory reserving )

**Reply by josegonzalez** [reply]
Have you checked out the documentation for the postgres service plugin? Specifically, search for the word "memory" in the readme for the plugin and you'll find references to it.

**Reply by dufok** [reply]
Found it: -m|--memory MEMORY: container memory limit in megabytes (default: unlimited)

I have load test's on my app with dokku-postgres and after 100users cpu starts be 100% (docker stats) but all CPU is maximum 20%...

**Reply by josegonzalez** [reply]
We don't limit CPU memory allocated to docker containers otherwise, so perhaps you have multiple cores and your postgres config is pegged to use only one of them?

**Reply by dufok** [reply]
Yes, it happened like that. I have 4 CPUs, but Postgres is using only one. In Docker stats, I saw 100% usage, but in monitoring it is only 25%. I want to use more CPU…
